### ISSUE TITLE
[BRIGHTEON] Videos truncated after 26 seconds

### ISSUE THREAD

**Issue by HalvorRaknes** [yt-dlp/yt-dlp#2876]
### Checklist

- [X] I'm reporting a broken site
- [X] I've verified that I'm running yt-dlp version **2022.02.04**. ([update instructions](https://github.com/yt-dlp/yt-dlp#update))
- [X] I've checked that all provided URLs are alive and playable in a browser
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/ytdl-org/youtube-dl#video-url-contains-an-ampersand-and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues including closed ones. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Norway

### Description

All videos (as far as I can determine) download but when played will only show the first 26 seconds.

When downloading with the app Video DownloadHelper the video downloads and plays without error.

### Verbose log

```shell
halvor@halvor-X551MA:~/Videos/Nyheter/Mike Adams/test$ yt-dlp -v -f hls-176 https://www.brighteon.com/1b6a9330-ecea-43d9-9f5f-29cc4af36e52
[debug] Command-line config: ['-v', '-f', 'hls-176', 'https://www.brighteon.com/1b6a9330-ecea-43d9-9f5f-29cc4af36e52']
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8, err utf-8, pref UTF-8
[debug] yt-dlp version 2022.02.04 [c1653e9ef]
[debug] Python version 3.8.10 (CPython 64bit) - Linux-5.4.0-100-generic-x86_64-with-glibc2.29
[debug] exe versions: ffmpeg 4.2.4, ffprobe 4.2.4, rtmpdump 2.4
[debug] Optional libraries: Cryptodome, secretstorage, mutagen, sqlite, websockets
[debug] Proxy map: {}
[debug] [generic] Extracting URL: https://www.brighteon.com/1b6a9330-ecea-43d9-9f5f-29cc4af36e52
[generic] 1b6a9330-ecea-43d9-9f5f-29cc4af36e52: Requesting header
WARNING: [generic] Falling back on generic information extractor.
[generic] 1b6a9330-ecea-43d9-9f5f-29cc4af36e52: Downloading webpage
[generic] 1b6a9330-ecea-43d9-9f5f-29cc4af36e52: Extracting information
[debug] Looking for video embeds
[generic] 1b6a9330-ecea-43d9-9f5f-29cc4af36e52: Downloading m3u8 information
[generic] 1b6a9330-ecea-43d9-9f5f-29cc4af36e52: Downloading MPD manifest
[debug] Identified a HTML5 media
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), acodec, filesize, fs_approx, tbr, vbr, abr, asr, proto, vext, aext, hasaud, source, id
[download] Downloading playlist: Dr. Yan Li-Meng interviewed by the Health Ranger on China's biological warfare threats against the world
[generic] playlist Dr. Yan Li-Meng interviewed by the Health Ranger on China's biological warfare threats against the world: Collected 1 videos; downloading 1 of them
[download] Downloading video 1 of 1
[info] 1b6a9330-ecea-43d9-9f5f-29cc4af36e52: Downloading 1 format(s): hls-176
[debug] Invoking downloader on "https://video.brighteon.com/file/Brighteon-staging/hls/b125b621-d766-406c-bfd5-6b3777d4f438_270.m3u8"
[download] Destination: Dr. Yan Li-Meng interviewed by the Health Ranger on China's biological warfare threats against the world [1b6a9330-ecea-43d9-9f5f-29cc4af36e52].mp4
[debug] ffmpeg command line: ffmpeg -y -loglevel verbose -headers 'User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.24 Safari/537.36
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8
Accept-Encoding: gzip, deflate
Accept-Language: en-us,en;q=0.5
Sec-Fetch-Mode: navigate
Referer: https://www.brighteon.com/1b6a9330-ecea-43d9-9f5f-29cc4af36e52
' -i https://video.brighteon.com/file/Brighteon-staging/hls/b125b621-d766-406c-bfd5-6b3777d4f438_270.m3u8 -c copy -f mp4 'file:Dr. Yan Li-Meng interviewed by the Health Ranger on China'"'"'s biological warfare threats against the world [1b6a9330-ecea-43d9-9f5f-29cc4af36e52].mp4.part'
ffmpeg version 4.2.4-1ubuntu0.1 Copyright (c) 2000-2020 the FFmpeg developers
  built with gcc 9 (Ubuntu 9.3.0-10ubuntu2)
  configuration: --prefix=/usr --extra-version=1ubuntu0.1 --toolchain=hardened --libdir=/usr/lib/x86_64-linux-gnu --incdir=/usr/include/x86_64-linux-gnu --arch=amd64 --enable-gpl --disable-stripping --enable-avresample --disable-filter=resample --enable-avisynth --enable-gnutls --enable-ladspa --enable-libaom --enable-libass --enable-libbluray --enable-libbs2b --enable-libcaca --enable-libcdio --enable-libcodec2 --enable-libflite --enable-libfontconfig --enable-libfreetype --enable-libfribidi --enable-libgme --enable-libgsm --enable-libjack --enable-libmp3lame --enable-libmysofa --enable-libopenjpeg --enable-libopenmpt --enable-libopus --enable-libpulse --enable-librsvg --enable-librubberband --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libssh --enable-libtheora --enable-libtwolame --enable-libvidstab --enable-libvorbis --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx265 --enable-libxml2 --enable-libxvid --enable-libzmq --enable-libzvbi --enable-lv2 --enable-omx --enable-openal --enable-opencl --enable-opengl --enable-sdl2 --enable-libdc1394 --enable-libdrm --enable-libiec61883 --enable-nvenc --enable-chromaprint --enable-frei0r --enable-libx264 --enable-shared
  WARNING: library configuration mismatch
  avcodec     configuration: --prefix=/usr --extra-version=1ubuntu0.1 --toolchain=hardened --libdir=/usr/lib/x86_64-linux-gnu --incdir=/usr/include/x86_64-linux-gnu --arch=amd64 --enable-gpl --disable-stripping --enable-avresample --disable-filter=resample --enable-avisynth --enable-gnutls --enable-ladspa --enable-libaom --enable-libass --enable-libbluray --enable-libbs2b --enable-libcaca --enable-libcdio --enable-libcodec2 --enable-libflite --enable-libfontconfig --enable-libfreetype --enable-libfribidi --enable-libgme --enable-libgsm --enable-libjack --enable-libmp3lame --enable-libmysofa --enable-libopenjpeg --enable-libopenmpt --enable-libopus --enable-libpulse --enable-librsvg --enable-librubberband --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libssh --enable-libtheora --enable-libtwolame --enable-libvidstab --enable-libvorbis --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx265 --enable-libxml2 --enable-libxvid --enable-libzmq --enable-libzvbi --enable-lv2 --enable-omx --enable-openal --enable-opencl --enable-opengl --enable-sdl2 --enable-libdc1394 --enable-libdrm --enable-libiec61883 --enable-nvenc --enable-chromaprint --enable-frei0r --enable-libx264 --enable-shared --enable-version3 --disable-doc --disable-programs --enable-libaribb24 --enable-liblensfun --enable-libopencore_amrnb --enable-libopencore_amrwb --enable-libtesseract --enable-libvo_amrwbenc
  libavutil      56. 31.100 / 56. 31.100
  libavcodec     58. 54.100 / 58. 54.100
  libavformat    58. 29.100 / 58. 29.100
  libavdevice    58.  8.100 / 58.  8.100
  libavfilter     7. 57.100 /  7. 57.100
  libavresample   4.  0.  0 /  4.  0.  0
  libswscale      5.  5.100 /  5.  5.100
  libswresample   3.  5.100 /  3.  5.100
  libpostproc    55.  5.100 / 55.  5.100
[tcp @ 0x55650cd3a9c0] Starting connection attempt to 104.17.166.78 port 443
[tcp @ 0x55650cd3a9c0] Successfully connected to 104.17.166.78 port 443
[hls @ 0x55650cd37880] Skip ('#EXT-X-VERSION:4')
[hls @ 0x55650cd37880] HLS request for url 'https://video.brighteon.com/file/Brighteon-staging/hls/b125b621-d766-406c-bfd5-6b3777d4f438_270.ts', offset 0, playlist 0
[hls @ 0x55650cd37880] Opening 'https://video.brighteon.com/file/Brighteon-staging/hls/b125b621-d766-406c-bfd5-6b3777d4f438_270.ts' for reading
[tcp @ 0x55650d022e40] Starting connection attempt to 104.17.166.78 port 443
[tcp @ 0x55650d022e40] Successfully connected to 104.17.166.78 port 443
[hls @ 0x55650cd37880] HLS request for url 'https://video.brighteon.com/file/Brighteon-staging/hls/b125b621-d766-406c-bfd5-6b3777d4f438_270.ts', offset 221464, playlist 0
[hls @ 0x55650cd37880] Opening 'https://video.brighteon.com/file/Brighteon-staging/hls/b125b621-d766-406c-bfd5-6b3777d4f438_270.ts' for reading
[tcp @ 0x55650d046ac0] Starting connection attempt to 104.17.166.78 port 443
[tcp @ 0x55650d046ac0] Successfully connected to 104.17.166.78 port 443
[h264 @ 0x55650d33fec0] Reinit context to 480x272, pix_fmt: yuv420p
Input #0, hls, from 'https://video.brighteon.com/file/Brighteon-staging/hls/b125b621-d766-406c-bfd5-6b3777d4f438_270.m3u8':
  Duration: 00:50:47.00, start: 1.999989, bitrate: 0 kb/s
  Program 0 
    Metadata:
      variant_bitrate : 0
    Stream #0:0: Video: h264 (Main), 1 reference frame ([27][0][0][0] / 0x001B), yuv420p(tv, bt709, left), 480x270 (480x272) [SAR 1:1 DAR 16:9], 14.99 fps, 14.99 tbr, 90k tbn, 29.97 tbc
    Metadata:
      variant_bitrate : 0
    Stream #0:1: Audio: aac (HE-AAC) ([15][0][0][0] / 0x000F), 48000 Hz, stereo, fltp
    Metadata:
      variant_bitrate : 0
Output #0, mp4, to 'file:Dr. Yan Li-Meng interviewed by the Health Ranger on China's biological warfare threats against the world [1b6a9330-ecea-43d9-9f5f-29cc4af36e52].mp4.part':
  Metadata:
    encoder         : Lavf58.29.100
    Stream #0:0: Video: h264 (Main), 1 reference frame (avc1 / 0x31637661), yuv420p(tv, bt709, left), 480x270 (0x0) [SAR 1:1 DAR 16:9], q=2-31, 14.99 fps, 14.99 tbr, 90k tbn, 90k tbc
    Metadata:
      variant_bitrate : 0
    Stream #0:1: Audio: aac (HE-AAC) (mp4a / 0x6134706D), 48000 Hz, stereo, fltp
    Metadata:
      variant_bitrate : 0
Stream mapping:
  Stream #0:0 -> #0:0 (copy)
  Stream #0:1 -> #0:1 (copy)
Press [q] to stop, [?] for help
Automatically inserted bitstream filter 'aac_adtstoasc'; args=''
[hls @ 0x55650cd37880] HLS request for url 'https://video.brighteon.com/file/Brighteon-staging/hls/b125b621-d766-406c-bfd5-6b3777d4f438_270.ts', offset 447252, playlist 0
[https @ 0x55650d020600] Opening 'https://video.brighteon.com/file/Brighteon-staging/hls/b125b621-d766-406c-bfd5-6b3777d4f438_270.ts' for reading
[hls @ 0x55650cd37880] HLS request for url 'https://video.brighteon.com/file/Brighteon-staging/hls/b125b621-d766-406c-bfd5-6b3777d4f438_270.ts', offset 672664, playlist 0
[https @ 0x55650d044100] Opening 'https://video.brighteon.com/file/Brighteon-staging/hls/b125b621-d766-406c-bfd5-6b3777d4f438_270.ts' for reading
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158237, current: 2083; changing to 2158238. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158238, current: 8089; changing to 2158239. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158239, current: 14095; changing to 2158240. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158240, current: 20101; changing to 2158241. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158241, current: 26107; changing to 2158242. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158242, current: 32113; changing to 2158243. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158243, current: 38119; changing to 2158244. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158244, current: 44125; changing to 2158245. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158245, current: 50131; changing to 2158246. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158246, current: 56137; changing to 2158247. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158247, current: 62143; changing to 2158248. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158248, current: 68149; changing to 2158249. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158249, current: 74155; changing to 2158250. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158250, current: 80161; changing to 2158251. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158251, current: 86167; changing to 2158252. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158252, current: 92173; changing to 2158253. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158253, current: 98179; changing to 2158254. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158254, current: 104185; changing to 2158255. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158255, current: 110191; changing to 2158256. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158256, current: 116197; changing to 2158257. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158257, current: 122203; changing to 2158258. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158258, current: 128209; changing to 2158259. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158259, current: 134215; changing to 2158260. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158260, current: 140221; changing to 2158261. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158261, current: 146227; changing to 2158262. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:1; previous: 1157120, current: 0; changing to 1157121. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:1; previous: 1157121, current: 2048; changing to 1157122. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:1; previous: 1157122, current: 4096; changing to 1157123. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:1; previous: 1157123, current: 6144; changing to 1157124. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158262, current: 152233; changing to 2158263. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158263, current: 158239; changing to 2158264. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:1; previous: 1157124, current: 8192; changing to 1157125. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:1; previous: 1157125, current: 10240; changing to 1157126. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:1; previous: 1157126, current: 12288; changing to 1157127. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:1; previous: 1157127, current: 14336; changing to 1157128. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158264, current: 164245; changing to 2158265. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158265, current: 170251; changing to 2158266. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158266, current: 176257; changing to 2158267. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:1; previous: 1157128, current: 16384; changing to 1157129. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:1; previous: 1157129, current: 18432; changing to 1157130. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:1; previous: 1157130, current: 20480; changing to 1157131. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:1; previous: 1157131, current: 22528; changing to 1157132. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158267, current: 182263; changing to 2158268. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158268, current: 188269; changing to 2158269. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:1; previous: 1157132, current: 24576; changing to 1157133. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:1; previous: 1157133, current: 26624; changing to 1157134. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:1; previous: 1157134, current: 28672; changing to 1157135. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:1; previous: 1157135, current: 30720; changing to 1157136. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158269, current: 194275; changing to 2158270. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158270, current: 200281; changing to 2158271. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2158271, current: 206287; changing to 2158272. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:1; previous: 1157136, current: 32768; changing to 1157137. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:1; previous: 1157137, current: 34816; changing to 1157138. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:1; previous: 1157138, current: 36864; changing to 1157139. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:1; previous: 1157139, current: 38912; changing to 1157140. This may result in incorrect timestamps in the output file.





[DELETED LINES]



[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:1; previous: 1233972, current: 808960; changing to 1233973. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2207580, current: 1653733; changing to 2207581. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2207581, current: 1659739; changing to 2207582. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2207582, current: 1665745; changing to 2207583. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:1; previous: 1233973, current: 811008; changing to 1233974. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:1; previous: 1233974, current: 813056; changing to 1233975. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:1; previous: 1233975, current: 815104; changing to 1233976. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:1; previous: 1233976, current: 817152; changing to 1233977. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2207583, current: 1671751; changing to 2207584. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2207584, current: 1677757; changing to 2207585. This may result in incorrect timestamps in the output file.
[mpegts @ 0x55650d01f240] PES packet size mismatch
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:1; previous: 1233977, current: 819200; changing to 1233978. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:1; previous: 1233978, current: 821248; changing to 1233979. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:0; previous: 2207585, current: 1683763; changing to 2207586. This may result in incorrect timestamps in the output file.
[mp4 @ 0x55650d338580] Non-monotonous DTS in output stream 0:1; previous: 1233979, current: 823296; changing to 1233980. This may result in incorrect timestamps in the output file.
No more output streams to write to, finishing.
frame=49709 fps=2027 q=-1.0 Lsize=   50794kB time=00:00:25.70 bitrate=16186.0kbits/s speed=1.05x    
video:11912kB audio:38772kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 0.216155%
Input file #0 (https://video.brighteon.com/file/Brighteon-staging/hls/b125b621-d766-406c-bfd5-6b3777d4f438_270.m3u8):
  Input stream #0:0 (video): 49709 packets read (12198355 bytes); 
  Input stream #0:1 (audio): 77426 packets read (39702906 bytes); 
  Total: 127135 packets (51901261 bytes) demuxed
Output file #0 (file:Dr. Yan Li-Meng interviewed by the Health Ranger on China's biological warfare threats against the world [1b6a9330-ecea-43d9-9f5f-29cc4af36e52].mp4.part):
  Output stream #0:0 (video): 49709 packets muxed (12198355 bytes); 
  Output stream #0:1 (audio): 77426 packets muxed (39702906 bytes); 
  Total: 127135 packets (51901261 bytes) muxed
[AVIOContext @ 0x55650d334840] Statistics: 24 seeks, 216 writeouts
[AVIOContext @ 0x55650d33fa00] Statistics: 33124252 bytes read, 0 seeks
[AVIOContext @ 0x55650d32e180] Statistics: 28125928 bytes read, 0 seeks
[AVIOContext @ 0x55650d01c600] Statistics: 22669 bytes read, 0 seeks
[ffmpeg] Downloaded 52013448 bytes
[download] 100% of 49.60MiB in 00:25
[download] Finished downloading playlist: Dr. Yan Li-Meng interviewed by the Health Ranger on China's biological warfare threats against the world
halvor@halvor-X551MA:~/Videos/Nyheter/Mike Adams/test$
```

**Comment by dirkf** [comment]
I think they mean 'non-monotonic'.

WFM (viewable to 50:46 in mpv-0.32) without those _ffmpeg_ diagnostics (ffmpeg 4.3.2):
* yt-dlp with the native HLS downloader and _ffmpeg_ fix-up
* yt-dlp with _ffmpeg_ as downloader
* yt-dl with _ffmpeg_ as downloader
* yt-dl with the native HLS downloader and _ffmpeg_ fix-up.

**Comment by flashdagger** [comment]
The HLS format seems to be a bit wonky. Here are your options:
* download HLS with native downloader (`--downloader native`)
* download MPD video and merge with audio (`-f 2+3`)

**Comment by pukkandan** [comment]
The default downloader in this case should have been `native` and not ffmpeg. This issue is accidentally fixed in 079a7cfc7191cc141ce09940e06ec897983526c3.

If you do want to use ffmpeg with it, there is nothing I can do about it. You'll just have to update your ffmpeg
